src/core/Table/Table.tsx

```diff
@@ -207,6 +218,8 @@ export const Table = <
     expanderCell,
     isRowDisabled,
     rowProps,
+    selectSubRows = true,
```

Comment:
In `react-table` its default value is treated as `true` but I wanted to set it explicitly.